src/imagery/i.eodag/i.eodag.py

```diff
@@ -228,10 +234,32 @@ def normalize_time(datetime_str: str):
     return normalized_datetime.isoformat()
 
 
+def no_fallback_search(search_parameters, provider):
+    try:
+        server_poke = dag.search(**search_parameters, provider=provider)
+        if server_poke[1] == 0:
+            gs.verbose(_("No products found"))
+            return SearchResult([])
+    except Exception as e:
+        gs.verbose(e)
+        gs.fatal(_("Server error, try again."))
```

Comment:
Oh yeah right, I should remove the "return" instead, because if I got into the exception, that means I couldn't connect to the server, so it should just terminate with an error message... will fix that.